src/ducks/files/Toolbar.jsx

```diff
@@ -29,6 +30,8 @@ class Toolbar extends React.Component {
 
   render () {
     const { t, disabled, displayedFolder, actions, onSelectItemsClick, uploadFiles } = this.props
+    const notRootfolder = displayedFolder !== undefined && displayedFolder !== null && displayedFolder.dir_id === ROOT_DIR_ID ? true : false
```

Comment:
You can write it that way:

```js
const notRootfolder = displayedFolder && displayedFolder.dir_id !== ROOT_DIR_ID
```

Be ware that here, you called the variable `notRootfolder` that is `true` if `displayedFilder.di_id` is the root folder id...